com.microsoft.mrtk.input/Interactors/HandJointInteractor.cs

```diff
@@ -44,7 +44,7 @@ public abstract class HandJointInteractor :
         /// at which the interaction occurs. This would be the tip of the index finger
         /// for a poke interactor, or some other computed position from other data sources.
         /// </summary>
-        protected abstract bool TryGetInteractionPoint(out HandJointPose jointPose);
+        protected abstract bool TryGetInteractionPoint(out Pose jointPose);
```

Comment:
nit: param naming here and anywhere it's overridden could be updated, since it's not really a "jointPose"

Reply:
Renamed!